### PR TITLE
Fix LayerNormalization on CPU

### LINKED ISSUE #46366: LayerNormalization crashes on empty inputs when run on CPU
**System information**
- Have I written custom code: Yes
- OS Platform and Distribution: Ubuntu 18.04
- TensorFlow installed from: binary
- TensorFlow version: 2.4.0
- Python version: 3.6.9

**Describe the current behavior**

`tf.keras.layers.LayerNormalization` crashes when the input is empty and the layer is executed on CPU.

**Describe the expected behavior**

The layer should not crash but return a tensor with the same shape.

**Standalone code to reproduce the issue**

```python
import os
os.environ["CUDA_VISIBLE_DEVICES"] = ""
import tensorflow as tf
layer = tf.keras.layers.LayerNormalization()
layer(tf.zeros([1, 0, 10]))
```

**Other info / logs**

The code above exits with this error:

```text
Floating point exception (core dumped)
```

### PR DESCRIPTION
**Pull request by around-star**
Fix #46366 . `LayerNormalization` layer crashes on empty inputs on CPU. This pull request would help `LayerNormalization` return the input as the output if the input value is `[]` .  This is a similar condition to `LayerNormalization on GPU` and `BatchNormalization` which returns back the empty input without throwing in an error.

### CODE REVIEW

**Comment by around-star** [comment]
@gbaned, can you please review this pull request or request someone else a review ? Thanks.
